tests/Text/KirbyTagsTest.php

```diff
@@ -66,4 +66,19 @@ public function testParseWithException()
 
         $this->assertSame('(test: foo)', KirbyTags::parse('(test: foo)'));
     }
+
+    public function testParseWithBrackets()
+    {
+        KirbyTag::$types = [
+            'test' => [
+                'html' => function ($tag) {
+                    return $tag->value;
+                }
+            ]
+        ];
+
+        $this->assertSame('foo(bar)', KirbyTags::parse('(test: foo(bar))'));
+        $this->assertSame('foo(bar)', KirbyTags::parse('(Test: foo(bar))'));
+        $this->assertSame('foo(bar)', KirbyTags::parse('(TEST: foo(bar))'));
```

Comment:
More tests would be great (e.g. with another argument following the main tag value, with text following the bracketed part inside the same argument, with multiple nested brackets inside the same value, with a value where opening and closing brackets don't match).

Reply:
I'm really hoping it will pass those tests - would be so cool! 😁

Reply:
Now all good 🚀